### ISSUE TITLE
Dark overlay in response pane of Terminal Chat reduces visibility

### ISSUE THREAD

**Issue by anth-git** [microsoft/terminal#18270]
### Windows Terminal version

1.23.3311.0

### Windows build number

10.0.22631.4460

### Steps to reproduce

Ask the chat a question

### Expected Behavior

The response pane in Terminal Chat should display text clearly without any dark overlay obstructing visibility

### Actual Behavior

![Image](https://github.com/user-attachments/assets/35712995-82f9-443f-a124-03f0d4726eb5)

**Comment by zadjii-msft** [comment]
Any chance your OS is in light theme while the Terminal is in dark theme (or any other difference in the theme like that?)

**Comment by anth-git** [comment]
> Any chance your OS is in light theme while the Terminal is in dark theme (or any other difference in the theme like that?)

Yes it was the case. When I switched to dark theme in Windows, it looks good.

![Image](https://github.com/user-attachments/assets/9805a6fe-6d2a-4a7b-9fc5-9b92cbc0246d)

**Comment by Burt-Silverman** [comment]
You may not wish to change the Windows theme. So, change the Terminal theme. Terminal->Settings->Appearance->Application Theme->Light.

**Comment by thedarkfalcon** [comment]
@Burt-Silverman thanks for the suggestion, but unfortunately this didn't fix the problem for me. Even when changing to the terminal light theme (and closing and reopening a tab) the Co-Pilot response text was still very dark and difficult to read.

![Image](https://github.com/user-attachments/assets/a723fa72-6075-413c-8d7a-8b1edf14f914)

Actually, just before posting this, I completely closed and re-opened Terminal and it resolved the problem.

![Image](https://github.com/user-attachments/assets/f58eb971-5e18-4179-94b9-318f9fc30923)

**Comment by Burt-Silverman** [comment]
I probably had to close and reopen Terminal, but I forgot to write that in my comment. Oops!

**Comment by robinmalik** [comment]
I was having this issue with Windows being in my preferred dark mode and Terminal in preferred light mode. I don't know if there will ever be a situation where we can have different settings and correct contrast, but I'll hope out hope!

**Comment by PankajBhojwani** [comment]
Closed via #19426
